crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -285,8 +285,9 @@ def restore_database
       end
 
       def proposal?(file)
-        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
-          file.to_s =~ /knife\/databags\/crowbar/
+        raw_data = JSON.parse(file.read)["raw_data"]
+        return false if raw_data.nil?
+        raw_data.has_key?("attributes") && raw_data.has_key?("deployment")
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. (https://github.com/bbatsov/ruby-style-guide#hash-key)